views/components/ChartList.jsx

```diff
@@ -9,14 +9,17 @@ import Button from '@material-ui/core/Button'
 import Link from '@material-ui/core/Link'
 import Delete from '@material-ui/icons/Delete'
 import Edit from '@material-ui/icons/Edit'
+import Share from '@material-ui/icons/Share'
 import PlaylistAdd from '@material-ui/icons/PlaylistAdd'
 
+import ShareChart from './ShareCharts'
 import { getCharts, deleteChart, duplicateChart } from '../fe-utils/fetchUtil'
 
 import { routes } from '../routes/routes'
 
-const ChartList = () => {
+const ChartList = ({ user }) => {
   const [chartList, setChartList] = useState([])
+  const [chartState, setChartState] = useState({})
```

Comment:
can we find a better name for this variable? It is very generically named but used for a specific purpose. What about something like `chartToShare`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/131/commits/4b7a005d45471fc07236b428f61a2f58b4284d6a